src/common/Filter.js

```diff
@@ -0,0 +1,24 @@
+import React, {Component} from 'react';
+import RadioButtons from './RadioButtons';
+import './styles/filter.less';
+
+export default class Filter extends Component {
+	constructor(props) {
+		super(props);
+		this.onChangeType = this.onChangeType.bind(this);
+	}
+
+	onChangeType(type) {
```

Comment:
I'd recommend you using an arrow function instead. You won't need to use .bind in the constructor then